transaction.go

```diff
@@ -236,6 +236,15 @@ func (tx *Transaction) EnsureParent() SpanID {
 	return tx.parentSpan
 }
 
+// ParentID returns the transaction's Parent ID generating a parent
+// span ID if one has not already been set and tx has not been ended.
+// If tx is nil or has been ended, a zero (invalid) SpanID is returned.
+//
+// Internally calls `Transaction.EnsureParent()`.
+func (tx *Transaction) ParentID() SpanID {
+	return tx.EnsureParent()
+}
```

Comment:
I'm not too keen on this -- EnsureParent is for the fairly specific, and uncommon use case of generating IDs for an upstream, such as RUM.

I think `ParentID` should behave the same both for spans and transactions: return a zero SpanID if there is no parent, otherwise return the parent ID.

Reply:
I think that makes more sense than what I initially did, by adding a new top level field `parentID` should reduce the amount of work this method does.